### DISCUSSION TITLE
Upgrade to version 0.9.40

### DISCUSSION THREAD

**Discussion by devdanco** [open-web3-stack/open-runtime-module-library#901] [General]
I have a question about the upgrade to 0.9.40. Do you know when it will happen? We're upgrading our node to the latest version of Polkadot, and I was wondering if you could tell me when the upgrade will be complete. Thank you

**Comment by xlc** [comment]
If happens when someone does it. Contributions are welcome!

**Reply by devdanco** [reply]
Ok. If I have a little bit of spare time I will do it.
